[DOC] Add CLI section to user documentation. (#12802)
Scille/parsec-cloud: docs/cli/workspace.rst

    ✔ Poll server for new certificates
    Found 1 workspace(s)
    0fe76194ad8649f6a8dc075020efaa4c - new_workspace: owner

Share
-----

This is to share a workspace to a user.
To get the workspace id and the user id use `workspace list` and `user list` (see :ref:`workspace list <cli_workspace_list>` and :ref:`user list <cli_user_list>` )
The possible roles (by most rights to less rights) are owner, manager, contributor, reader.

.. code-block:: shell

    parsec-cli workspace share -d b1e1 -w 0fe76194ad8649f6a8dc075020efaa4c -u 25d3bfbc-3738-4a37-ae20-84ad67b423cc -r manager
    Enter password for the device:
    Workspace has been shared

List user
---------

To list users that have access to the given workspace

.. code-block:: shell

    parsec-cli workspace list-users -d b1e1 -w 0fe76194ad8649f6a8dc075020efaa4c
    Enter password for the device:
    ✔ Poll server for new certificates
    Workspace 0fe76194-ad86-49f6-a8dc-075020efaa4c is shared with 2 user(s)
    • User b35ae5d5-1734-42e6-802a-39024d25399d (ADMIN) - Alice (alice@example.com) has role owner
    • User 25d3bfbc-3738-4a37-ae20-84ad67b423cc (STANDARD) - Bob (bob@example.com) has role manager

Import
------

> To import a local file to a workspace.

Reviewer: The `import` command is much more complex: it can import files or folders, it can create parent directories and overwrite (or fail) if destination file exists.

IMO it is worth to explain all these features here.

https://github.com/Scille/parsec-cloud/blob/73a768e0a31a02798b3d25069a0dbf66dda8b81f/cli/src/commands/workspace/import.rs#L17-L35

.. code-block:: shell

    parsec-cli workspace import -d b1e1 -w 0fe76194ad8649f6a8dc075020efaa4c README.md /README.md
    Enter password for the device:

> Archive

Reviewer: ```suggestion
Archive or Delete
```

-------

To change the accessibility status of the workspace:

> - available: the default status, the workspace can be edited

Reviewer: ```suggestion
- available: the workspace can be edited and shared normally
```

> - archived: the workspace is read only

Reviewer: ```suggestion
- archived: the workspace is still accessible to its members but in read-only
```

> - deletion-planned-in-seconds: the workspace will be deleted in the specified amount of seconds

Reviewer: I would mention that there is a minimum number of seconds defined by the organization that needs to be respected.

BTW what happen if the specified number of seconds is lower? does the command fail? does it show a clear error message?

Author: if the delay is too small it fails with that error : `Updating workspace archiving statusError: Archiving period is too short`

.. code-block:: shell

    parsec-cli workspace archive -d b1e1 -w 0fe76194ad8649f6a8dc075020efaa4c --archived
    Enter password for the device:

>     Workspace archiving status has been updated

Reviewer: Maybe is better to give examples for each option for this particular commmand

```shell
# Archive the workspace
$ parsec-cli workspace archive -d b1e1 -w 0fe76194ad8649f6a8dc075020efaa4c --archived 
...

# Schedule for deletion in 10 minutes
$ parsec-cli workspace archive -d b1e1 -w 0fe76194ad8649f6a8dc075020efaa4c --p --deletion-planned-in-seconds 600
...

# Restore the workspace back to available
$ parsec-cli workspace archive -d b1e1 -w 0fe76194ad8649f6a8dc075020efaa4c --p --available
```
